webpack.config.js

```diff
@@ -1,21 +1,23 @@
 const ESLintPlugin = require('eslint-webpack-plugin');
 const TerserPlugin = require('terser-webpack-plugin');
 const CopyPlugin = require('copy-webpack-plugin');
+const webpack = require('webpack');
 const path = require('path');
 const fs = require('fs');
 
 const optimization = {
   minimizer: [
-    new TerserPlugin({
-      cache: true,
-      parallel: true,
-      terserOptions: {
-        compress: false,
-        ecma: 6,
-        mangle: false,
-      },
-      sourceMap: true,
```

Comment:
I believe that `devtool: 'source-map'` handles the `sourceMap: true` from looking at the latest terser-webpack-plugin documentation.
```
[webpack-cli] Invalid options object. Terser Plugin has been initialized using an options object that does not match the API schema.
54
 - options has an unknown property 'sourceMap'. These properties are valid:
55
   object { test?, include?, exclude?, terserOptions?, extractComments?, parallel?, minify? }
```